test/bdd/fixtures/kms/docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
       - KMS_KEY_MANAGER_STORAGE_PREFIX=authzkmskm
       - KMS_HUB_AUTH_URL=https://hub-auth.example.com:8070
       - KMS_HUB_AUTH_API_TOKEN=test_token
+      - KMS_CACHE_EXPIRATION=10m
```

Comment:
This env variable is missing from your start command - need to add it there and pass it in to your cache provider

Reply:
support for that env was implemented in [my previous PR](https://github.com/trustbloc/hub-kms/pull/128) when caching for EDV was added, here I am reusing that functionality for Hub Auth calls as well

Reply:
Oh, I see - so it in this branch the env key doesn't exists, but in the master branch it does? (So when merged, it'll be there?)

Reply:
My mistake - I had checked out the wrong branch locally (there was one in the upstream that's similar but not quite the right onw)